### PR TITLE
Remove unused dependencies for sdk-wallet and sdk-network-providers

### PR DESCRIPTION
**Pull request by danielailie**
#### Did you test the changes locally ?
- [X] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [x] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/sdk-and-tools/sdk-js/sdk-js.md [hunk 1]
```diff
@@ -17,9 +17,7 @@ Base libraries:
...
-| [sdk-core](https://www.npmjs.com/package/@multiversx/sdk-core)                           | [Github](https://github.com/multiversx/mx-sdk-js-core)              | Basic components for interacting with the blockchain and with smart contracts. |
-| [sdk-wallet](https://www.npmjs.com/package/@multiversx/sdk-wallet)                       | [Github](https://github.com/multiversx/mx-sdk-js-wallet)            | Core wallet components (generation, signing).                                  |
-| [sdk-network-providers](https://www.npmjs.com/package/@multiversx/sdk-network-providers) | [Github](https://github.com/multiversx/mx-sdk-js-network-providers) | Network Provider (API, Gateway) components (compatible with sdk-js).           |
+| [sdk-core](https://www.npmjs.com/package/@multiversx/sdk-core)                           | [Github](https://github.com/multiversx/mx-sdk-js-core)              | The `sdk-core` package is a unification of the previous packages (multiversx/sdk-wallet and multiversx/sdk-network-providers into sdk-core). It has basic components for interacting with the blockchain and with smart contracts. |
```
Comment (andreibancioiu) [inline docs/sdk-and-tools/sdk-js/sdk-js.md:20]: :rocket:

Comment (andreibancioiu) [inline docs/sdk-and-tools/sdk-js/sdk-js.md:20]: Names of packages can be all within `code snippet`. :upside_down_face:

File: docs/sdk-and-tools/sdk-py/sdk-py.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ This SDK consists of a CLI tool, namely `mxpy`, and Python helpers and utilities
 
 [comment]: # (mx-context-auto)
 
-## `sdk-py`: the Python libraries
+## `sdk-py`: the Python library
```
Comment (andreibancioiu) [inline docs/sdk-and-tools/sdk-py/sdk-py.md:32]: :rocket: :snake:

File: docs/sdk-and-tools/sdk-py/sdk-py.md [hunk 3]
```diff
@@ -38,6 +38,3 @@ The `multiversx-sdk` package is a unification of the previous packages (multiver
 | Package                                                                                     | Source code                                                               | Description                                                                                   |
 |---------------------------------------------------------------------------------------------|---------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------|
 | [multiversx-sdk](https://pypi.org/project/multiversx-sdk)                           | [mx-sdk-py](https://github.com/multiversx/mx-sdk-py)            | The all in one Python SDK for interacting with MultiversX. |
```
Comment (andreibancioiu) [inline docs/sdk-and-tools/sdk-py/sdk-py.md:40]: `all-in-one` :clown_face:

**Review by andreibancioiu — DISMISSED** [review]

**Review by popenta — APPROVED** [review]

**Review by popenta — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**